units_cpp/src/units/headers/units/constants/planetary_rotations.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * planetary_rotations.h
+ *
+ *  Created on: Nov 3, 2017
+ *      Author: oriash93
+ */
+
+#ifndef SRC_UNITS_HEADERS_UNITS_CONSTANTS_PLANETARY_ROTATIONS_H_
+#define SRC_UNITS_HEADERS_UNITS_CONSTANTS_PLANETARY_ROTATIONS_H_
+
+#include <units/duration.h>
+
+namespace units {
+namespace planetary_rotations
+{
+
+constexpr duration::Seconds sunRotationPeriod{2192831.6};
+constexpr duration::Seconds mercuryRotationPeriod{5067030};
+constexpr duration::Seconds venusRotationPeriod{-20996760};
```

Comment:
why the minus?

Reply:
It's something called [retrograde motion](https://en.wikipedia.org/wiki/Retrograde_and_prograde_motion), it means it's moving in an opposite direction.
Notice Venus and Uranus are in negative units [here](https://en.wikipedia.org/wiki/Rotation_period#Rotation_period_of_selected_objects).

Maybe I misunderstood.

Reply:
Cool! didn't know that.

then it's ok :+1: